### DISCUSSION TITLE
How to add an attribute to the input element of Select

### DISCUSSION THREAD

**Discussion by chilltemp** [mantinedev/mantine#153] [Q&A] [answered]
When using the new `<Select />` element, how can I add an attribute to the `<input />` it creates?  I add `data-lpignore="true"` to all the inputs that aren't user/pass/address, etc.  Otherwise LastPass adds it's form-fill icon where it doesn't provide any value (and this annoys me, probably more than it should).

**Comment by cstrat** [comment]
Not sure but would `wrapperProps` work?

`<Select wrapperProps={{ "data-lpignore": "true" }} ... />`

Does last-pass respect the `autoComplete="off`" flag on the `<form>` element? I use that to stop the browser auto-completion from showing.

**Reply by chilltemp** [reply]
I'll try both of these tomorrow.  Thanks

**Reply by chilltemp** [reply]
Lastpass ignores the `autoComplete` attribute by default.

**Comment by rtivital** [comment, marked as answer]
I guess this is missing from documentation, but you can just set these props on component and it should pass it to input – https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/components/Select/Select.tsx#L319

So it should be `<Select data-lpignore="true" />`

`wrapperProps` adds props to root element (InputWrapper component)

**Reply by chilltemp** [reply]
This is actually what I tried first, and it didn't work.  What did I learn?  Lastpass + webpack hotreload = confused developer.  I did a hard reload after adding the attribute, and it worked.

p.s.  Nice work on this lib!  Your source code taught me the `Props extends DIV` technique that I'm now using in my components.
